packages/component-library/draft/Kaizen/Tabs/styles.scss

```diff
@@ -1,52 +1,75 @@
 @import "~@kaizen/design-tokens/sass/color";
+@import "~@kaizen/design-tokens/sass/border";
 @import "~@kaizen/component-library/styles/border";
 @import "~@kaizen/component-library/styles/color";
 @import "~@kaizen/component-library/styles/layout";
 @import "~@kaizen/component-library/styles/type";
 
 .container {
   @include ca-margin($bottom: $ca-grid);
-  border-bottom: 1px solid $ca-border-color;
+  border-bottom: 1px solid add-alpha($kz-color-wisteria-700, 10%);
   display: flex;
 }
 
 .tab {
-  $underline-height: 4px;
-
-  @include ca-padding($top: $ca-grid * 0.5, $bottom: $ca-grid * 0.75);
   @include ca-type-ideal-button;
   @include ca-inherit-baseline;
+
+  position: relative;
+  top: 0; // this overrides baseline shift. @include ca-inherit-baseline does not work because
+  // we need to make this position: relative, for the absolutely positioned underline.
+  height: 3 * $ca-grid;
+  min-width: 4.5rem; // give tabs with short text some breathing room
   text-decoration: none;
   color: $kz-color-wisteria-800;
-  // This border will become visible when the .activeTab class is applied
-  border-bottom: $underline-height solid transparent;
-  // This invisible border is to compensate for the bottom border,
-  // as to make sure the content is vertically centered
-  border-top: $underline-height solid transparent;
+  padding: 0 ($ca-grid * 0.5);
   cursor: pointer;
 
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  border-top-left-radius: $kz-border-borderless-border-radius;
+  border-top-right-radius: $kz-border-borderless-border-radius;
+
   &:hover {
     color: $kz-color-cluny-500;
     text-decoration: none;
+    background-color: $kz-color-cluny-100;
   }
 }
 
 .tab + .tab {
-  @include ca-margin($start: $ca-grid);
+  @include ca-margin($start: $ca-grid * 0.5);
 }
 
 .activeTab {
   composes: tab;
   color: $kz-color-cluny-500;
-  border-bottom-color: currentColor;
+
+  &::after {
+    $underline-height: 5px;
+    content: "";
+    display: block;
+    border-top-left-radius: $underline-height;
+    border-top-right-radius: $underline-height;
+    background-color: $kz-color-cluny-500;
+    height: $underline-height;
+    width: 100%;
+    position: absolute;
+    left: 0;
+    right: 0;
+    bottom: 0;
+  }
 }
 
 .disabledTab {
   composes: tab;
-  color: $kz-color-wisteria-300;
   cursor: inherit;
+  opacity: 0.3;
```

Comment:
I think this generally a good way to make things disabled visually, and the Zen UI Kit seems to do it like this everywhere. We might want to consider adding a token for it somewhere. Out of scope for this though.